packages/blockchain-tools/src/interfaces/CrossmarkWallet.ts

```diff
@@ -25,8 +25,9 @@ export default class CrossmarkWallet extends XrplCommon {
 
   async sendPayment({ address, amount, memo }: { address: string; amount: number; memo?: string }) {
     console.log('PAY', address, amount, memo)
+    /* TODO: FIX
```

Comment:
If crossmark isn't working, we shouldn't have it enabled in appConfig

Reply:
There was a small issue with the amount type but for now it is not working so I will remove it from the config